scrap.js

```diff
@@ -31,5 +31,9 @@ async function fetchOrgsWithLeaders() {
 
 ;(async () => {
   const data = await fetchOrgsWithLeaders()
+  // sort data by completed_task_instance_count
```

Comment:
I was meant to say to make the double empty lines to a single one, sorry.

Please do keep the previous empty lines, except that double ones, make it into a single line.